CMakeLists.txt

```diff
@@ -4,34 +4,44 @@ cmake_policy(SET CMP0077 NEW)
 cmake_minimum_required(VERSION 3.14)
 project(hsm VERSION 0.2.0 LANGUAGES CXX)
 
-include(cmake/CPM.cmake)
-CPMAddPackage(NAME tiny_tuple
-  GITHUB_REPOSITORY "APokorny/tiny_tuple"
-  GIT_TAG development)
-CPMAddPackage("gh:TheLartians/PackageProject.cmake@1.6.0")
+include(cmake/get_cpm.cmake)
+CPMAddPackage(
+    NAME tiny_tuple
```

Comment:
why the change in indent?

Reply:
I increased the indent from 2 to 4 spaces for readability, I also did the same on tiny_tuple